scipy/signal/lti_conversion.py

```diff
@@ -459,6 +465,27 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
         cd = c
         dd = d
 
+    elif method == 'foh':
```

Comment:
You can get a more concise representation if you directly form `em`. I've implemented something similar here

https://github.com/ilayn/harold/blob/master/harold/_discrete_funcs.py#L156